postgres-external-storage/src/main/java/com/netflix/conductor/postgres/storage/PostgresPayloadStorage.java

```diff
@@ -92,12 +110,22 @@ public ExternalStorageLocation getLocation(
     public void upload(String key, InputStream payload, long payloadSize) {
         try (Connection conn = postgresDataSource.getConnection();
                 PreparedStatement stmt =
-                        conn.prepareStatement("INSERT INTO " + tableName + " VALUES (?, ?)")) {
+                        conn.prepareStatement(
+                                "INSERT INTO "
+                                        + tableName
+                                        + " (id, data) VALUES (?, ?) ON CONFLICT(id) "
+                                        + "DO UPDATE SET created_on=CURRENT_TIMESTAMP")) {
             stmt.setString(1, key);
             stmt.setBinaryStream(2, payload, payloadSize);
-            stmt.executeUpdate();
-            LOGGER.debug(
-                    "External PostgreSQL uploaded key: {}, payload size: {}", key, payloadSize);
+            if (stmt.executeUpdate() == 0) {
+                LOGGER.warn(
```

Comment:
Is this really a warning ? should we care ? If so then we should add some details to the log message as to why it was not updated

Reply:
Maybe it is not so important, but it can be a good way how to check if an insert/update was successful https://stackoverflow.com/questions/24378270/how-do-you-determine-if-an-insert-or-update-was-successful-using-java-and-mysql. Maybe we can change it to debug so in case of some problem we can find out if the insert/update was successful?